com.github.tchx84.Flatseal.json

```diff
@@ -0,0 +1,40 @@
+{
+    "app-id": "com.github.tchx84.Flatseal",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.tchx84.Flatseal",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=/var/lib/flatpak/app:ro",
+        "--filesystem=~/.local/share/flatpak/app:ro",
+        "--filesystem=~/.local/share/flatpak/overrides"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "flatseal",
+            "builddir": true,
```

Comment:
IIRC it's enabled by default for meson.

Reply:
It came with Builder's template hah, but I can remove it.